src/IO/ITKImageInputFileFormat.cxx

```diff
@@ -164,6 +159,36 @@ ITK_coordinates_to_STIR_physical_coordinates
   return stir_coord;
 }
 
+/* Convert an ITK Pixel (i.e., float) to a STIR Pixel. */
+typename STIRImageSingle::pixel_type
+ITK_pixel_to_STIR_pixel(typename ITKImageSingle::PixelType itk_pixel,
+                        const STIRImageSingle &stir_image,
+                        bool is_displacement_field=false)
```

Comment:
to avoid a compiler warning, just say `bool` (as the argument isn't used)

Reply:
It seems I still need this now that the second function is a specialisation so that I can define the default of false.

Reply:
With latest changes and the other version being a speicialisation, I think this is now needed again so that I can define the default.